web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -351,19 +352,23 @@ const EXTERNAL_SERVICES = [
     maxsize: Infinity,
     endpoint: 'https://neurosift.app?p=/nwb&url=$asset_dandi_url$&dandisetId=$dandiset_id$&dandisetVersion=$dandiset_version$', // eslint-disable-line max-len
   },
-
   {
     name: 'Neurosift',
     regex: /\.nwb\.lindi\.(json|tar)$/,
     maxsize: Infinity,
     endpoint: 'https://neurosift.app?p=/nwb&url=$asset_dandi_url$&st=lindi&dandisetId=$dandiset_id$&dandisetVersion=$dandiset_version$', // eslint-disable-line max-len
   },
-
   {
     name: 'Neurosift',
     regex: /\.avi$/,
     maxsize: Infinity,
     endpoint: 'https://neurosift.app?p=/avi&url=$asset_dandi_url$&dandisetId=$dandiset_id$&dandisetVersion=$dandiset_version$', // eslint-disable-line max-len
+  },
+  {
+    name: 'Neuroglancer',
+    regex: /\.nii(\.gz)?$|\.zarr$/,
+    maxsize: Infinity,
+    endpoint: '', // defaults to redirectNeuroglancerUrl logic
```

Comment:
I think we should aim for adjusting the "schema" here to reflect this new need -- that we need

```suggestion
    item_handler: 'redirectNeuroglancerUrl',
```

and then use that in the service code?